src/main/java/com/walmartlabs/x12/asn856/DefaultAsn856TransactionSetParser.java

```diff
@@ -517,5 +545,34 @@ private void doItemSegments(X12Segment segment, SegmentIterator segmentIterator,
                 break;
         }    
     }
+    
+    /**
+     * handle the segment lines that are part of the Batch (appearing before the next
+     * HL loop)
+     * 
+     * @param segment
+     * @param segmentIterator
+     * @param items
+     */
+    private void doBatchSegments(X12Segment segment, SegmentIterator segmentIterator, Batch batch) {
+        switch (segment.getIdentifier()) {
+            case PIDProductIdentification.IDENTIFIER:
+                batch.addPIDProductIdentification(PIDPartyIdentificationParser.parse(segment));
+                break;
+            case LINItemIdentification.IDENTIFIER:
+                batch.setItemIdentifications(LINItemIdentificationParser.parse(segment));
```

Comment:
Should we do `addLINItemIdentification` here?

Reply:
per discussion fixing on next PR

Reply:
the LINItemIdentificationParser will return a list of all the item/product ids on the segment line 
that is why we use the setItemIdentifications method here

if there was more than one LIN segment in the same Batch loop then we would only get the last one
but there is only supposed to be one